packages/ember-cli-mirage/index.js

```diff
@@ -106,8 +106,8 @@ module.exports = {
     });
   },
 
-  treeForApp(appTree) {
-    let trees = [appTree];
+  treeForAddon(addonTree) {
```

Comment:
@mansona could you please explain a bit this change?

I got the point about initializer location in app folder and that totally makes sense. But this change seems separate?